src/RemoteNetwork.cpp

```diff
@@ -13,6 +13,7 @@
 #include <QTimeZone>
 #include <QtNetwork/QNetworkProxy>
 #include <json.hpp>
+#include <QRegExp>
```

Comment:
Why this? It seems incoherent with QRegExp being replaced by QRegularExpression for porting to Qt6.

Reply:
**QRegularExpression** does not supports **indexIn** and I did not want to rewrite
```cpp
   QRegExp regex("^.*modification-date=\"(.+)\";.*$");
            regex.setMinimal(true); // Set to non-greedy matching
            if(regex.indexIn(content_disposition) != -1)
                last_modified = QDateTime::fromString(regex.cap(1), Qt::ISODate);
```

For Qt6 QRegExp is provided by **Core5Compat** and it's not included by default

Reply:
Is this of any help? 
> replace if (rx.indexIn( by QRegularExpressionMatch match = rx.match(searchString); if (match.hasMatch()) and update the invalid code parts

From https://github.com/Komet/MediaElch/issues/1086

Reply:
I want to drop **Core5Compat**but, there are some edge cases with **QRegExp** and **TextCodecs** and I am not confident enough to rewrite it (especially regex see CrowdStrike :) ).

Reply:
Ok, if Core5Compat is working, no problem on my part. For me, it could be merged when you consider it ready.